packages/client/src/client.ts

```diff
@@ -86,6 +86,7 @@ import { MsgInitiateTokenDeposit } from "./codegen/opinit/ophost/v1/tx";
 
 export const SKIP_API_URL = "https://api.skip.build";
 
+export const GAS_STATION_CHAIN_IDS = ["bbn-test-5", "bbn-1", "11155111"];
```

Comment:
you can't use `GAS_STATION_CHAIN_IDS`, because `11155111` was a hack to skip gas validation